src/main/resources/templates/company.html

```diff
@@ -1,51 +1,46 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
 <head>
-  <link rel="stylesheet" href="index.css">
+  <title>Voluntier</title>
```

Comment:
The `<head>` tag must have a `<title>` attribute defined to be considered valid HTML. [Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title)